prijateli_tree/app/routers/games.py

```diff
@@ -205,6 +211,23 @@ def get_previous_answers(
 ###############################
 
 
+@router.get("/session/{session_id}")
+def route_session_access(
+    request: Request, session_id: int, db: Session = Depends(get_db)
+):
+    # Do some logic things
```

Comment:
Here you can do what you need to do for directing people to the games.

Reply:
What do you have in mind by this? Like we'd have the facilitators open this page on various computers and "log students in" here?

Reply:
There's a ready field attached to every `GameSessionPlayer` object, and I presumed you'd use that to make sure everyone is prepared to play. Once everyone is ready you send them to the first game. You could also ignore the `ready` on each game and just assume that once they are in the session they're always ready.